README.md

```diff
@@ -0,0 +1,9 @@
+# Open Wallet
+A work in progress - Experiment in which we attempt to manage invoices, expenses, budgets and analysis in a transparent manner.
+
+## .env
+```MONGODB_URI``` Path to reach your database (e.g.```mongodb+srv://<username>:<password>@cluster0-e7fdv.mongodb.net/test?retryWrites=true```) <br />
```

Comment:
`<br>` doesn't work in markdown :)

You can either use 2 spaces at the end of a line to break to the next, or do a full line break for a new paragraph.